scss/components/_folder.scss

```diff
@@ -1,11 +1,16 @@
 @import "./button";
 @import "./mixins/flex-mix.scss";
 @import "./src/fonts";
+@import "../variables";
 
 %column {
-  width: 372px;
+  width: 31%;
```

Comment:
Bad value

Reply:
Agree, but unfortunately it's the only way to make this almost PixelPerfect(